components/post_view/post_message_view/post_message_view.jsx

```diff
@@ -174,7 +175,7 @@ export default class PostMessageView extends React.PureComponent {
                     tabIndex='0'
                     id={id}
                     className='post-message__text'
-                    onClick={Utils.handleFormattedTextClick}
+                    onClick={(e) => Utils.handleFormattedTextClick(e, this.props.currentRelativeTeamUrl)}
```

Comment:
As a preference we don't use arrow functions when adding it to handlers for this reason https://stackoverflow.com/a/48740930/2902013. Can you change this to be  `onClick={this.handleFormattedTextClick}` and the define the method on the class?